### PR TITLE
Update migration guide

### PR DESCRIPTION
**Pull request by phanimarupaka**
This PR updates the migration guide.

### CODE REVIEW

File: site/installation/migration.md [hunk 1]
```diff
@@ -327,8 +327,8 @@ kpt `v0.39`) to `v1` version(compatible with kpt `v1.0`).
 [openapi validations]: https://googlecontainertools.github.io/kpt/guides/producer/setters/#openapi-validations
 [required setters]: https://googlecontainertools.github.io/kpt/guides/producer/setters/#required-setters
 [auto-setters]: https://googlecontainertools.github.io/kpt/concepts/setters/#auto-setters
-[migrating inventory objects]: https://googlecontainertools.github.io/kpt/reference/cli/live/alpha/
```
Comment (frankfarzan) [inline site/installation/migration.md:330]: change the wording for this link, it's not a "detailed Guide".

Comment (phanimarupaka) [reply site/installation/migration.md:330]: updated.

**Review by frankfarzan — APPROVED** [review]

File: site/installation/migration.md [hunk 2]
```diff
@@ -70,12 +70,12 @@ directory by default.
...
 | `kpt pkg update LOCAL_PKG_DIR[@VERSION] [flags]`                                                  | `kpt pkg update [PKG_PATH][@version] [flags]` <br> Pulls in upstream changes and merges them into a local package.                                                                                                                                   |
-| `kpt pkg fix DIR [flags]`                                                                         | `kpt fn eval --image gcr.io/kpt-fn/fix:v0.2` <br> Fix a local package which is using deprecated features.                                                                                                                                        |
+| `kpt pkg fix DIR [flags]`                                                                         | `kpt fn eval --image gcr.io/kpt-fn/fix:v0.2` <br> Fix a local package which is using deprecated features.                                                                                                                                            |
```
Comment (droot) [inline site/installation/migration.md:73]: I think we should include `--include-meta-resources` or `-m` here.

**Review by droot — CHANGES_REQUESTED** [review]
Looks good, a minor suggestion around `fix`

File: site/installation/migration.md (hunk 2 above)
Comment (phanimarupaka) [reply site/installation/migration.md]: Done.

**Review by droot — APPROVED** [review]

**Pull request merged**